mod.template.json

```diff
@@ -7,9 +7,10 @@
   "packageId": "com.beatgames.beatsaber",
   "packageVersion": "1.19.0",
   "description": "Core mod required for other mods to function!",
+  "coverImage": "cover.png",
   "dependencies": [],
   "modFiles": [],
   "libraryFiles": [],
   "fileCopies": [],
   "copyExtensions": []
-}
```

Comment:
This is a bad change and you should feel sorry for yourself and i hate you and everything that you stand for, please try again later, this pr is horrible and i hate everything about it please see a mental health institute